docusaurus/docs/dev-docs/upgrade-tool.md

```diff
@@ -25,6 +25,23 @@ The upgrade tool is a Strapi package and can be run from the CLI.
 
 <DoNotMigrateYet />
 
+## Scope
+
+While the upgrade tool helps upgrade apps and plugins, it doesn't cover every aspect of it. 
+
+:white_check_mark: The upgrade tool supports:
+- Updating the project's dependencies
+- Applying automatic code transformation to existing files
+- Installing or re-installing the correct dependencies for the project
+
+:x: The upgrade tool doesn't support:
+- Modifying the file tree by adding, removing or moving files and directories
+- Migrating the app's data. This is handled by Strapi database migrations
```

Comment:
```suggestion
- Migrating the application's data. This is handled by Strapi database migrations.
```

Reply:
That's funny, I was asked to replace application with app :sweat_smile: